packages/fluentui/docs/src/examples/components/Dialog/Variations/DialogExampleZoomContent.shorthand.tsx

```diff
@@ -0,0 +1,55 @@
+import * as React from 'react';
+import * as _ from 'lodash';
+import { Button, Dialog } from '@fluentui/react-northstar';
+import { useBooleanKnob } from '@fluentui/docs-components';
+import { CloseIcon } from '@fluentui/react-icons-northstar';
+
+const DialogExampleZoomContent: React.FC = () => {
+  const [open, setOpen] = useBooleanKnob({ name: 'open' });
+  const dialogVariables = {
+    rootWidth: '550px',
+  };
+  return (
+    <Dialog
+      open={open}
+      onOpen={() => setOpen(true)}
+      onCancel={() => setOpen(false)}
+      onConfirm={() => setOpen(false)}
+      variables={dialogVariables}
+      cancelButton="Cancel"
+      confirmButton="Confirm"
+      content={{
+        content: (
+          <>
+            {_.times(5, i => (
+              <p key={i}>
+                Lorem ipsum dolor sit amet, consectetur adipiscing elit. Praesent ornare, neque eget egestas fermentum,
+                massa risus mollis orci, et ullamcorper purus turpis at risus. Fusce pharetra mollis sapien nec commodo.
+                Quisque ut congue sem, vel aliquam augue. Vestibulum ante ipsum primis in faucibus orci luctus et
+                ultrices posuere cubilia Curae; Duis porttitor, nunc quis dapibus imperdiet, ligula sem egestas orci,
+                sed volutpat ipsum felis vitae velit. Sed maximus egestas dui elementum aliquam. In hac habitasse platea
+                dictumst. Proin maximus nibh velit, ut ornare dui mollis viverra.
+              </p>
+            ))}
+          </>
+        ),
+        styles: {
```

Comment:
why are these styles only in the example? Can we make it easier for the consumers to apply them, either using a prop or a variable? Or are they specific to this example?

Reply:
From what I understand the default breakpoints and styles might not suite all content. So the examples using styles are actually override guides

Reply:
in that case, maybe we should prepare different alternatives? Because - as all dialogs will need to be responsive and support zoom, it would not be practical to ask all consumers to provide these styles for all of their dialogs.

Reply:
it's hard to say, it really really depends on content. I don't even know where to start with a 'good' set of alternatives.

I guess you're thinking about something like small, medium, large breakpoints at 400% zoom ?

Reply:
I moved styles with @media to dialogStyles.ts
But I don't think that it's possible to avoid styles overrides from outside. 

For example, for long content max-height should be defined outside. Same with some custom footer.